charts/opentelemetry-operator/UPGRADING.md

```diff
@@ -1,5 +1,9 @@
 # Upgrade guidelines
 
+## 0.72.0 to 0.72.1
+
+Prior to 0.72.1, feature gates could be enabled via the `manager.featureGates` property. As feature gates may require extra configuration to work properly, e.g. deploying extra permissions on the ClusterRole, the chart has been updated to make use of the `manager.featureGatesMap` property which allows the chart to smartly configure feature gates. If the `manager.featureGatesMap` property is set, the old `manager.featureGates` property will be ignored.
```

Comment:
Lets add an entry to NOTES.txt to promote that `featureGates` is deprecated and users should use `featureGatesMap`.

Also a failure scenario for users configuring both `featureGates` and `featureGatesMap`. I don't want to deal with merging them so lets allow only 1 to be set.

Reply:
Done